tests/unit/reports/test_editable.py

```diff
@@ -1674,8 +1674,7 @@ def test_delete_session(self, sample_report):
         assert res["report"] == expected_result["report"]
         assert res["totals"] == expected_result["totals"]
         assert res == expected_result
-        report.delete_session(0)
-        report.delete_session(2)
+        report.delete_multiple_sessions({0, 2})
```

Comment:
Nice, was going to ask if there was a test testing multiple sessions, had only seen a dict with 1, tyty